src/test/kotlin/com/sourcegraph/cody/chat/PromptHistoryTest.kt

```diff
@@ -0,0 +1 @@
+
```

Comment:
? :)

Reply:
just a missing newline added automatically

Reply:
Why is this file needed at all?

Reply:
ah, sorry - my bad. I misunderstood the diff

Reply:
removed